modules/photon/system-part1/linker.ld

```diff
@@ -3,7 +3,7 @@ MEMORY
     APP_FLASH (rx)  : ORIGIN = 0x08020000, LENGTH = 256K
 
     /* todo - SRAM must start also at an offset after what has been reserved for system-module 1 */
-    SRAM      (rwx) : ORIGIN = 0x20000000, LENGTH = 384
+    SRAM      (rwx) : ORIGIN = 0x20000000, LENGTH = 1K
```

Comment:
Sure, I'll check that. Actually, SRAM usage increased by only 300 bytes if I remember correctly, I just put "1K" to have some more space than necessary.